src/frontend/planner_test/tests/testdata/expr.yaml

```diff
@@ -472,9 +472,13 @@
       ├─StreamDynamicFilter { predicate: (t.v1 >= now), output: [t.v1, t.v2, t._row_id] }
       | ├─StreamTableScan { table: t, columns: [t.v1, t.v2, t._row_id], pk: [t._row_id], dist: UpstreamHashShard(t._row_id) }
       | └─StreamExchange { dist: Broadcast }
-      |   └─StreamNow { output: [now] }
+      |   └─StreamProject { exprs: [now], watermark_columns: [now] }
+      |     └─StreamShare { id = 94 }
+      |       └─StreamNow { output: [now] }
       └─StreamExchange { dist: Broadcast }
-        └─StreamNow { output: [now] }
+        └─StreamProject { exprs: [now], watermark_columns: [now] }
+          └─StreamShare { id = 94 }
+            └─StreamNow { output: [now] }
```

Comment:
We shouldn't share Now as well right?

Reply:
We can share the common sub-plan with the following properties:
1. include at least one table scan.
2. Not just simply a table scan.